frappe/core/doctype/activity_log/activity_log.py

```diff
@@ -48,6 +48,7 @@ def clear_activity_logs(days=None):
 	if not days:
 		days = 90
 	doctype = DocType("Activity Log")
+	duration = (Now() - Interval(days=days))
 	frappe.db.delete(doctype, filters=(
-		doctype.creation < PseudoColumn(f"({Now() - Interval(days=days)})")
+		doctype.creation < duration
```

Comment:
```suggestion
		doctype.creation < (Now() - Interval(days=days))
```

We don't need to assign a separate variable for this. ^ has lesser cognitive load